### ISSUE TITLE
[Dataset records] delete dataset records

### ISSUE THREAD

**Issue by frascuchon** [recognai/rubrix#706]
Sometimes we could clean a rubrix dataset. The current workflow should be load desired records and log in a new dataset and then delete the old dataset.

It would be nice a "delete by query" action to clean dataset records. What do you think @dvsrepo @dcfidalgo ?

**Comment by dcfidalgo** [comment]
Yes, I imagine that this could be a useful feature. But also one with great power and with great power comes great responsibility ... :hankey: Not sure if we could somehow do this in two steps where you have the possibility to review your query, something in that direction.

**Comment by frascuchon** [comment]
You can always do a `rb.load` with same query and make your stuff
